WcaOnRails/app/views/admin/anonymize_person.html.erb

```diff
@@ -38,8 +38,24 @@
             <li>Request data anonymization from WCA Forum by <%= mail_to Team.wac.leader.user.email[0], name = "Emailing the WAC Leader", cc: Team.wrt.email, subject: "Anonymization Request", body: "Hello " + Team.wac.leader.user.name + ",
   WCA has received an anonymization request for #{@anonymize_person.account.email}. Can you please search active users ( https://forum.worldcubeassociation.org/admin/users/list/active) for any users using this email. If there are any can you please anonymize the user's data.
   Thanks - WCA Results Team" %>.</li>
+            <br>
+            <li>Request data removal from OAuth Access Grants.</li>
+            <% access_grants = @anonymize_person.account.oauth_access_grants %>
```

Comment:
Don't forget to filter for `revoked_at` not nil, as the user may already have some older applications they revoked before requesting anon

Reply:
Done!